components/dot_menu/dot_menu.tsx

```diff
@@ -50,7 +51,7 @@ type Props = {
     handleDropdownOpened: (open: boolean) => void;
     handleAddReactionClick?: () => void;
     isMenuOpen?: boolean;
-    isReadOnly?: boolean;
+    isReadOnly?: boolean | null;
```

Comment:
I had to amend this attribute's type because fixed component uses it with `null` value in some cases. Cf. components/search_results_item/search_results_item.tsx:375